src/OpenTracing/NoopTracer.php

```diff
@@ -15,7 +15,7 @@ public static function create()
 
     public function startSpan(
         $operationName,
-        SpanReference $parentReference = null,
+        Reference $parentReference = null,
```

Comment:
`$parentReference` already implies a parent-child relationship, it should just be `$reference`